### ISSUE TITLE
Images are not accessible to assistive technology

### ISSUE THREAD

**Issue by steverep** [whitesource/merge-confidence#1650]
The images used for age, adoption, passing, and confidence level are not accessible to screen readers. Currently, the markdown looks like:

```
![age](...)
```

The text in the square brackets should be useful alternative text. Just repeating the title is not describe the meaning of the image.
The markdown simply needs to be adjusted to reflect actual age, percentage, or confidence level.

**Comment by rarkins** [comment]
Unfortunately I'm not sure this is achievable.

Renovate inserts the markdown with the help of this preset, and it is static.

Then the image itself is rendered dynamically from a separate backend.

Even if it were possible for Renovate to insert actual values into the markdown:
- Such values could only be adjusted any time Renovate runs (so they would often be "wrong"/contradicting what's in the image)
- It would mean PR content might be updated 100s of times in their life as the values change. The PRs would show up as edited/"last updated" recently at most times

This sounds like a shared problem for all types of badges inserted into markdown. Are you aware of any solutions where badges in Pull Requests have their text updated dynamically?

**Comment by steverep** [comment]
The SVG generated is actually already accessible with appropriate text (they include `role="img" aria-label="..."` and also have a `<title>`).  The problem is that info cannot make it into the document using an `<img>` element.  It needs to instead be embedded using `<object>`, `<iframe>`, or possibly even nested in another `<svg>`.  I'm not sure which, if any, of those GH markdown will allow.

**Comment by rarkins** [comment]
> @rarkins can you explain why you, or your team, picked images instead of text?

Plain text within PR bodies would be inefficient because it requires the PR body to be updated every time the value changes. e.g. initially a new version would have 0% adoption, then 1%, then 2%, etc. The PR would be "out of date" regularly, and then needing to be edited potentially dozens of times. Instead, images can be updated. Pretty standard reason why people use badges.

Same applies to the original request - Renovate would need to increase its API use on GitHub dramatically and run much more often in order to keep values updated.
